data/hai/hai reveal 6 end.txt

```diff
@@ -0,0 +1,665 @@
+# Copyright (c) 2020 MasterOfGrey
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+########################################################################
+
+mission "Hai Reveal [D01] Cleanup"
+	landing
+	name "Hai Diplomacy"
+	description "With the crisis now averted, all that remains is to go to Mountaintop for the formalities."
+	source "Hai-home"
+	destination "Mountaintop"
+	passengers 4
+	blocked "You need <bunks> passenger spaces to transport Xilin, Alondo, Sayari and Teeneep."
+	to offer
+		has "Hai Rescue [C08] Freedom: done"
+	on offer
+		event "mountaintop ascendant"
+		fail "Hai Rescue [C07A] Devil-Hide Fleet"
+		log `Returned the Hai to their own space to bring an end to the missing Hai saga. All that remains is to work out what to do about the leak, and the cover story so thin as to provide no modesty to its wearers.`
```

Comment:
I'm not so sure about that, since the description following it only applies to the cover story.

Reply:
It's grouped as " ' what to do ' " about 'the leak' and 'the cover story' [,which is] 'so thin as to...' "

This is when remembering the vocabulary behind this stuff would actually be beneficial lol

The two part list doesn't get broken up by a comma because it's all part of the same phrase (and the greater clause), but the description on the one part after should be separated, especially with the implied words being dropped.

I can't remember the name of the construction, but it's essentially the same as saying "Jon, my uncle," or even "my uncle Jon," because when the construction is nothing but a proper name in it's entirety then you -don't- separate it off.